docs/guides/functions.ipynb

```diff
@@ -440,7 +414,49 @@
   },
   {
    "cell_type": "markdown",
-   "id": "9137071a-e39c-4b66-afbd-697db38cc670",
+   "id": "af7fa72a-f272-43ff-800d-5ae37dd42b2b",
+   "metadata": {},
+   "source": [
+    "To retrieve a job's job_id by its position in the list, use `catalog.get_job_by_id()`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "id": "18d87f07-8a8e-4b93-80d0-7c07e49272e2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Get the job_id for the job at index position 1\n",
+    "latest_job = old_jobs[1]\n",
```

Comment:
I'm not sure why we're using index 1? The latest job would be index 0

```suggestion
    "# Get the last job that's executed1\n",
    "latest_job = old_jobs[0]\n",
```

Reply:
oh right - I can change that. I had wanted to do index 1 in my previous iterations, but now that we're using latest_job as the variable, I'll change it